charts/llm-d-modelservice/values.yaml

```diff
@@ -58,6 +58,31 @@ accelerator:
       - name: VLLM_WORKER_MULTIPROC_METHOD
         value: "spawn"
 
+dra:
+  enabled: false  # true: use this block instead of the `accelerator`
+  type: nvidia
+  claimTemplates:
+  - name: nvidia
+    class: gpu.nvidia.com
+    match: "exactly"
+    count: 1
+    selectors: {}
```

Comment:
Is it necessary to enumerate all types? Could we just write them in the comments? :-)